README.rst

```diff
@@ -0,0 +1,115 @@
+PyCraft
+=======
+
+A community driven fork of `foglemans "Minecraft"
+repo <https://github.com/fogleman/Minecraft>`_.
+
+|#pycraft on freenode|
+
+
+Motivation
+----------
+
+The original project had great ideas and implemented really cool things,
+but it was primarily designed around teaching.  This project is intended
+to create a community-driven engine for complicated voxel/roguelike
+games which ultimately can use and extend
+`GURPS <https://en.wikipedia.org/wiki/GURPS>`__
+
+    The Generic Universal RolePlaying System, or GURPS, is a tabletop
+    role-playing game system designed to allow for play in any game
+    setting.
+
+This project will be very permissive in accepting pull-requests.
+
+
+Screenshot
+----------
+
+.. figure:: screenshot.png
+   :alt:
+
+Virtual Environment (Recommended)
+---------------------------------
+
+.. code:: bash
+
+    # create a virtual environment
+    virtualenv -p python3 ~/.venv/pycraft # (or wherever)
+    # you may need to add execute permissions
+    chmod -R a+x ~/.venv
+    # activate
+    . ~/.venv/pycraft/bin/activate # on mac
+    . ~/.venv/pycraft/Scripts/activate # on windows
+    # deactivate (when you're done)
+    deactivate
+
+Installing
+----------
+
+.. code:: bash
+
+    pip install -e .
+
+**option 1:**
+
+.. code:: bash
+
+    pip install -e .[dev]
+    # or: python3 setup.py develop
```

Comment:
On the versioning tag:  I just haven't found a tool that can automate exactly what I want.  I usually use git-flow, so I have a stable/release branch (master) and an unstable/integration branch (develop).  I generally use a '-dev' on the end of my version until I submit to master.  I don't have (nor do I really want) control/ownership over this particular repository, or I would have setup a git-flow branching scheme as well.